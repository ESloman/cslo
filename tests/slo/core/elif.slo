
for (var x = 0; x < 10; x++) {
    if (x == 2) {
        println("x is 2");
    } elif (x == 1) {
        println("x is 1");
    } elif (x == 3) {
        println("x is 3");
    } else {
        println("x is neither");
    }
}


var y = 0;
while (y < 10) {
    if (y == 0) {
        println("y is 0");
        println(y);
    } elif (y % 2 == 0) {
        println("y is even");
        println(y);
    } else {
        println("y is odd");
    }
    y += 1;
}

for (var z = 0; z < 3; z++) {
    if (z == 1) {
        println("z is 1");
    } elif (z == 2) {
        println("z == 2");
    }
}

println("end of elif.slo");
