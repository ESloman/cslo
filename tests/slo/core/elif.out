x is neither
x is 1
x is 2
x is 3
x is neither
x is neither
x is neither
x is neither
x is neither
x is neither
y is 0
0
y is odd
y is even
2
y is odd
y is even
4
y is odd
y is even
6
y is odd
y is even
8
y is odd
z is 1
z == 2
end of elif.slo
